common/persistence/sql/history_store.go

```diff
@@ -464,11 +464,16 @@ func (m *sqlExecutionStore) GetAllHistoryTreeBranches(
 }
 
 // GetHistoryTree returns all branch information of a tree
```

Comment:
This comment also needs to be fixed, as well as in `cassandra/hisotry_store.go` (on the very top of this PR).

Reply:
Yup, just fixed.